app/models/null_chapter.rb

```diff
@@ -1,3 +1,3 @@
 class NullChapter < NullObject
-  def short_code(_ = {}); end
+  def short_code; end
```

Comment:
Not needed?

https://github.com/trade-tariff/trade-tariff-backend/blob/main/app/lib/null_object.rb#L10-L12